### PR TITLE
Linter warning fixes for a couple of files, #507

### PR DESCRIPTION
**Pull request by knod**

### CODE REVIEW

**Comment by knod** [comment]
@wacii: I'll take both of those, thank you very much! Maybe we'll wait till after this PR is merged, though.

**Comment by knod** [comment]
I'll look into implementation 👍

File: src/forms/BenefitsTable.js [hunk 1]
```diff
@@ -73,36 +78,41 @@ const BenefitsTable = function (props) {
...
 
 
-
-
-
-
   const SNAPBenefitRow = function(props){
-    if (!client.current.hasSnap) {return (null);}
+    if (!client.current.hasSnap) {
+      return (null);
```
Comment (ethanbb) [inline src/forms/BenefitsTable.js]: Why have parentheses around return values?

Comment (knod) [reply src/forms/BenefitsTable.js]: Not sure if I did that or if it was someone else. If it was me, I was hallucinating at the time. Maybe it was to mimic the way that Components are usually returned in parens. Not sure, though.

**Pull request merged**